apis/runs.py

```diff
@@ -19,9 +19,10 @@ def create_run():
     Returns the user ID of the run created.
     '''
 
-    query = "INSERT INTO `runs` (`prompt_id`,`user_id`) VALUES (%s, %s)"
+    query = "INSERT INTO `runs` (`start_time`,`prompt_id`,`user_id`) VALUES (%s, %s, %s)"
```

Comment:
Just clarifying, this is reversing the changes made here correct? 

https://github.com/wikispeedruns/wikipedia-speedruns/commit/7ddb2aefc5e4b2af283b86aba77db43514fb74ae#diff-6ec42f2a7a7aed120b17a97db67f4e3b3079fb84c024b09fa30d99ca3d4be359L22

Since this functionality was already supported

Reply:
Yeah